### DISCUSSION TITLE
Problems logging in with 2FA

### DISCUSSION THREAD

**Discussion by Xmetalfanx** [community/community#65947] [New to GitHub]
I finally added 2FA and not thinking I'd have issues, i was fine with that.   The problem is no matter what I try the auth. code always fails ... i can login fine with my initial (7 logins left apparently) recovery codes, but that is running out.   

I have tried different auth. apps/addons (I would like to store any info i need in Bitwarden then i can login to that) .... I am not sure what I am doing wrong.   I even had issues setting things up but on a phone i have (without a phone plan, but it can get on wifi) I think i had to do an extra time sync or something and then it finally worked.   in some auth. apps/addons I do not see any option for that.

I dont mnid 2FA or an extra step (I rather not HAVE TO HAVE an extra device with me everywhere) ... but this is baffling me

**Comment by OmolloRovy** [comment]
One possible cause of 2FA failure is a time synchronization issue. 2FA applications use time-based one-time passwords (TOTP) that are time-sensitive. If the time on your device is not synchronized correctly, it can lead to authentication failures. Ensure that the time on both your device and the server where you are authenticating (e.g., GitHub) is accurate and synchronized via an internet time server. Some authentication apps have a time sync option, so look for that and enable it if available.
